src/olympia/abuse/models.py

```diff
@@ -1091,7 +1091,7 @@ def get_action_helper(self, *, overridden_action=None, appealed_action=None):
         if appealed_action:
             # target appeal
             if appealed_action in DECISION_ACTIONS.REMOVING:
-                if self.action in DECISION_ACTIONS.APPROVING:
+                if self.action in DECISION_ACTIONS.NON_OFFENDING:
```

Comment:
it _shouldn't_ happen, but we can't stop it - at the point we process we can't force the moderator in Cinder to choose an explicitly reviewed policy - so we either ignore as a possiblity because we don't allow it (i.e. no email to the developer, and no un-disable of content); or handle it as best we can.